File: BIOS_SAI VISHAL/Tasks/Set-1/Programming/C/chall-1.c
#include <stdio.h>
int main() {
    int a,b,c,n,t;
    printf("no.of.test cases:");
    scanf("%d",&t);
    while(t>0)
    {printf("enter the 1st number:");
     scanf("%d",&a);
     printf("enter the 2nd number:");
     scanf("%d",&b);
     printf("which nth number do you want:");
     scanf("%d",&n);
     while(n>1)
     {  c=a^b;
        a=b;
        b=c;
        n=n-1;
     }
     printf("nth number is %d\n",c);
     t=t-1;
    }
}

File: BIOS_SAI VISHAL/Tasks/Set-1/Programming/Python/chall-1.py
N=int(input("enter the number of dungeons:"))
list=[]
for i in range(0, N):
    l= int(input("type(0 and 1):"))
    list.append(l)
    N=N-1
n=len(list)
sum=0
for i in range(0,n,1):
    if list[i]==1:
        p=0
        sum=sum+1
        while p<=n-1:
              if list[p]==0:
                   list[p]=1
              else:
                   list[p]=0
              p=p+1
print(sum)
